android/sdk/src/main/res/values/donottranslate.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <string name="pref_file_name" translatable="false">OrganicMapsPrefs</string>
+  <string name="pref_enable_logging" translatable="false">EnableLogging</string>
```

Comment:
Why is it non-translatable?

Reply:
Because it's not needed to be translatable
It's a key